src/opm/output/eclipse/AggregateMSWData.cpp

```diff
@@ -225,15 +231,17 @@ namespace {
     }
 
     Opm::RestartIO::Helpers::SegmentSetFlowRates
-    getSegmentSetFlowRates(const std::string& wname, const Opm::WellSegments& segSet, const std::vector<Opm::data::Connection>& rateConns,
-                           const Opm::WellConnections& welConns, const Opm::UnitSystem& units)
+    getSegmentSetFlowRates(const Opm::WellSegments& segSet,
+                           const std::vector<Opm::data::Connection>& rateConns,
+                           const Opm::WellConnections& welConns,
+                           const Opm::UnitSystem& units)
     {
         std::vector<double> sofr (segSet.size(), 0.);
         std::vector<double> swfr (segSet.size(), 0.);
         std::vector<double> sgfr (segSet.size(), 0.);
         //
         //call function to calculate the individual segment source/sink terms
-        auto sSSST = getSegmentSetSSTerms(wname, segSet, rateConns, welConns, units);
+        auto sSSST = getSegmentSetSSTerms(segSet, rateConns, welConns, units);
```

Comment:
Navnet skjemmer ingen - men detta var mange 'S'?

Reply:
> detta var mange 'S'?

Storholt, Stenshjemmet, Stensen and Sjøbrend...

It's of course the [source/sink term](https://github.com/OPM/opm-common/blob/8d3e4191626b65e23656e7237e4f76c0782d7920/opm/output/eclipse/AggregateMSWData.hpp#L47) structure.  I suppose we could rename the variable to the less accurate `segmentSources` to add some context to its use.